pkg/stanza/operator/helper/transformer.go

```diff
@@ -94,10 +96,22 @@ func (t *TransformerOperator) ProcessWith(ctx context.Context, entry *entry.Entr
 
 // HandleEntryError will handle an entry error using the on_error strategy.
 func (t *TransformerOperator) HandleEntryError(ctx context.Context, entry *entry.Entry, err error) error {
+	logFields := []zap.Field{
+		zap.Any("error", err),
+		zap.Any("action", t.OnError),
+	}
+	if entry != nil {
```

Comment:
Does this actually happen? Seems like if the entry is nil we might as well just bail immediately.

Reply:
Good point. I don't think it would happen and even if it did things would break in other places first because I don't see nil checks on the entry anywhere. 👍